schema/event/createEventStep2.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { date, number, object, string } from "yup"
+
+export const CreateEventStep2Schema = object({
+  startDate: date()
```

Comment:
Must check if user select date from past (I have it written in one of my schema already)

Reply:
right.. forgot that. ty